### DISCUSSION TITLE
How to download and load .bin files?

### DISCUSSION THREAD

**Discussion by swamyg** [oobabooga/text-generation-webui#4713] [Q&A]
I'm looking for help to download and load this model: https://huggingface.co/Amod/falcon7b-fine-tuned-therapy-merged
It's not in GGUF format. I see there are two bin files. Not sure how to download and run these in text-generation-webui

Is there a guide or wiki someone can link to?

**Comment by HiroseKoichi** [comment]
There are two ways you can download it: either manually download all the files listed and put them inside a folder, or use TextGen WebUI's built-in model downloader. If you want to use the model downloader, which I recommend, then copy the name of the user who uploaded the model along with the name of the model and separate them with a /. For example, Amod/falcon7b-fine-tuned-therapy-merged would be what you paste into 'Download model or LoRA' in the model tab. There's a button at the top of each model page that allows you to easily copy this information to your clipboard. Unquantized models can be used with the transformers loader.
